lib/config/presets/internal/monorepo.ts

```diff
@@ -276,6 +276,7 @@ const patternGroups = {
   clarity: ['^@cds/', '^@clr/'],
   embroider: '^@embroider/',
   fullcalendar: '^@fullcalendar/',
+  rjsf: '^@rjsf/',
```

Comment:
why not adding via sourceUrl like most of the others?

Reply:
Only because I wasn't sure of the difference! I'm happy to change it over.